src/mon/OSDMonitor.cc

```diff
@@ -8685,7 +8685,13 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
      mempool::osdmap::map<string,map<string,string>> old_ec_profiles =
        osdmap.get_erasure_code_profiles();
      auto ec_profiles = pending_inc.get_erasure_code_profiles();
-     ec_profiles.merge(old_ec_profiles);
+#ifdef HAVE_STDLIB_MAP_SPLICING
+     ec_profiles.insert(old_ec_profiles);
```

Comment:
shouldn't this have been `ec.profiles.merge()`?

Reply:
@jecluis 
Sharp eye...
Thanx